db/migrate/20180829130513_update_kexec_uri.rb

```diff
@@ -0,0 +1,28 @@
+class UpdateKexecUri < ActiveRecord::Migration[5.0]
+  def update_templates
+    ["Discovery Red Hat kexec", "Discovery Debian kexec"].each do |template_name|
+      if t = ProvisioningTemplate.unscoped.find_by_name(template_name)
+        contents = t.template.gsub(/@kexec_kernel/, '@kernel_uri').gsub(/@kexec_initrd/, '@initrd_uri')
+        t.update_attributes(:template => contents)
```

Comment:
Btw. `update_attributes` is skipping the audit creation. Given we would need to fix it by `t.save!`, which has a risk of us hitting some issues with invalid user data (Using a fake model is not an option either, as the audited would use the fake's class name to store the audit.

I guess it's not worth the effort and we could just call `without_auditing` and not caring about the audits at all here.